pkg/drain/nodegroup.go

```diff
@@ -144,18 +145,31 @@ func NodeGroup(clientSet kubernetes.Interface, ng *api.NodeGroup, waitTimeout ti
 
 			for _, node := range nodes.Items {
 				if newPendingNodes.Has(node.Name) {
+					retry := 1
 					pending, err := evictPods(drainer, &node)
-					if err != nil {
-						return err
```

Comment:
I think all that's need is `logger.Warning` here instead of return, followed by `time.Sleep`.